staging/src/k8s.io/kubectl/pkg/util/prune/prune.go

```diff
@@ -68,7 +68,7 @@ func GetRESTMappings(mapper meta.RESTMapper, pruneResources []Resource, namespac
 		// TODO in kubectl v1.29, add back non-namespaced resource only if namespace is not specified
 		pruneResources = append(pruneResources, defaultNonNamespacedPruneResources...)
 		if namespaceSpecified {
-			klog.Warning("Deprecated: kubectl apply will no longer prune non-namespaced resources by default when used with the --namespace flag in a future release. To preserve the current behaviour, list the resources you want to target explicitly in the --prune-allowlist flag.")
+			klog.InfoS("Deprecated: kubectl apply will no longer prune non-namespaced resources by default when used with the --namespace flag in a future release. To preserve the current behaviour, list the resources you want to target explicitly in the --prune-allowlist flag.")
```

Comment:
Is this a user-visible change? If it is, we should update the changelog entry from `NONE`.

Reply:
Also, why is this no longer a warning?

Reply:
Structured logging only has "error" and "info" log messages (https://github.com/go-logr/logr#why-not-named-levels-like-infowarningerror).

Reply:
Changed the code to contextual logging, this comment is outdated.